scripts/register.js

```diff
@@ -0,0 +1,11 @@
+const esbuild = require('esbuild')
+const fs = require('fs')
+require.extensions['.ts'] = (mod, filename) => {
+  const ts = fs.readFileSync(filename, 'utf-8')
+  const { code } = esbuild.transformSync(ts, {
+    loader: 'ts',
+    target: 'es2017',
+    format: 'cjs',
+  })
+  mod._compile(code, filename)
```

Comment:
:speech_balloon: hah, is this basically a homegrown `ts-node` using ESBuild? Nice :)

Reply:
yes, but much faster than ts-node

Reply:
There will be some complex logic, and I'm not comfortable with JS so I convert build file to TS